modules/ipi-install-creating-a-disconnected-registry.adoc

```diff
@@ -185,6 +185,38 @@ $ jq ".auths += $AUTHSTRING" < pull-secret.txt > pull-secret-update.txt
 $ sudo scp kni@provisioner:/usr/local/bin/oc /usr/local/bin
 ----
 
+. Set the required environment variables:
+.. Export the release version:
++
+[source,terminal]
+----
+$ VERSION=<release_version>
+----
++
+For `<release_version>`, specify the tag that corresponds to the version of {product-title} to
+install, such as `{product-version}`.
```

Comment:
Can you put this on one line?

Consider something like this, which might be a bit clearer:

`For `<release_version>`, specify the tag that matches the version of {product-title} you want to install. For example,`{product-version}`.